lib/react-loading.jsx

```diff
@@ -8,8 +8,14 @@ export default class Loading extends Component {
     color: PropTypes.string,
     delay: PropTypes.number,
     type: PropTypes.string,
-    height: PropTypes.number,
-    width: PropTypes.number,
+    height: PropTypes.oneOfType([
```

Comment:
Looks good. Could you please update `readme.md` accordingly. I think we need 1-2 examples to make this clear because i can see that many people get confused by these 2 props values. What do you think?